### PR TITLE
Made the is_uploaded_file() check configurable

### PR DESCRIPTION
**Pull request by burzum**
Optional is_uploaded_file() validation.

### CODE REVIEW

**Comment by markstory** [comment]
Why?

**Comment by burzum** [comment]
@markstory tried to use this in a shell?

File: src/Validation/Validation.php [hunk 1]
```diff
@@ -1165,6 +1165,7 @@ public static function uploadedFile($file, array $options = [])
             'maxSize' => null,
             'types' => null,
             'optional' => false,
+            'isUploadedFile' => true
```
Comment (robertpustulka) [inline src/Validation/Validation.php]: IMO this suggests that you're telling a file is uploaded and a validator actually shouldn't check it.

Maybe `checkUploadedFile` would better describe what it does?

Comment (burzum) [inline src/Validation/Validation.php]: Exactly. How about `checkIsUploadedFile`?

**Comment by markstory** [comment]
@burzum This config option makes it possible to do remote file manipulation though. If it is in an application, an bad person can manipulate the POST data to trick application code into thinking that local files have been 'uploaded'.

**Comment by burzum** [comment]
@markstory it sounded to me like you agree to it when we've discussed it on Slack. OK, then let's close it and I stick to overriding it in my application.

**Pull request closed without merging**